sdk/src/integrationTest/java/IntegrationTestEnv.java

```diff
@@ -173,6 +173,7 @@ public void nextNode(Map<String, AccountId> outMap) throws Exception {
                     outMap.put(node.getKey(), node.getValue());
                     return;
                 } catch (Exception ignored) {
+                    throw ignored;
```

Comment:
This is an oxymoron. Shouldn't even catch it if you're just going to re-throw it as is.